lib/updatePlist.js

```diff
@@ -45,9 +44,40 @@ var getPlistConfigValue = function(plistConfig, manifest, key) {
   return value;
 };
 
+
+var replaceObjectRecursively = function (plistObj, manifest, plistConfig, key) {
+  var value;
+
+  Object.keys(plistConfig).forEach(function (nested_key) {
+    value = getPlistConfigValue(plistConfig, manifest, nested_key);
```

Comment:
wont it break if plistConfig[key] is object ?

Reply:
No.It won't checked for values.